docs/source/index.rst

```diff
@@ -0,0 +1,51 @@
+.. InnerEye documentation master file, created by
+   sphinx-quickstart on Sun Jun 28 18:04:34 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+InnerEye-DeepLearning Documentation
+===================================
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Overview and user guides
+
+   md/README.md
+   md/environment.md
+   md/WSL.md
+   md/hello_world_model.md
+   md/setting_up_aml.md
+   md/creating_dataset.md
+   md/building_models.md
+   md/sample_tasks.md
+   md/bring_your_own_model.md
+   md/debugging_and_monitoring.md
+   md/model_diagnostics.md
+   md/move_model.md
+   md/hippocampus_model.md
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Further reading for contributors
+
+   md/pull_requests.md
+   md/testing.md
+   md/contributing.md
+   md/deploy_on_aml.md
+   md/fastmri.md
+   md/innereye_as_submodule.md
+   md/releases.md
+   md/self_supervised_models.md
+   md/CHANGELOG.md
```

Comment:
does `md/CHANGELOG.md` exist? I don't see it in the folder structure here

Reply:
I think it's copied in `docs/source/conf.py`.

Reply:
@peterhessey, why did you go back to copying instead of symlinking?

Reply:
Symlinking was causing some peculiar errors in a couple of the build pipelines where they're checking file sizes for reasons unknown :( It also makes it possible to edit paths in the files without changing your local source code.